.github/workflows/ci.yml

```diff
@@ -43,6 +43,24 @@ jobs:
         run: |
           cargo test --no-run --all-targets
 
+  doc:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        rust: [stable, beta, nightly, 1.77]
+        os: [ubuntu-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: install toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ matrix.rust }}
+      - name: Build docs
+        run: |
+          cargo doc
```

Comment:
Down the line, I'd suggest also setting `RUSTDOCFLAGS` (env var) to `-Dwarnings`. I'm guessing now it won't pass yet?